pkg/dockerregistry/testutil/fakeopenshift.go

```diff
@@ -37,75 +38,80 @@ func NewFakeOpenShift() *FakeOpenShift {
 // the stateful fake in-memory OpenShift reactors. The fake OpenShift is
 // available for direct interaction, so you can make buggy states.
 func NewFakeOpenShiftWithClient() (*FakeOpenShift, *testclient.Fake) {
-	os := NewFakeOpenShift()
+	fos := NewFakeOpenShift()
 	client := &testclient.Fake{}
-	os.AddReactorsTo(client)
-	return os, client
+	fos.AddReactorsTo(client)
+	return fos, client
 }
 
-func (os *FakeOpenShift) CreateImage(image *imageapi.Image) (*imageapi.Image, error) {
-	_, ok := os.imagesStorage[image.Name]
+func (fos *FakeOpenShift) CreateImage(image *imageapi.Image) (*imageapi.Image, error) {
+	_, ok := fos.imagesStorage[image.Name]
 	if ok {
-		return nil, errors.NewAlreadyExists(unversioned.GroupResource{
-			Group:    "",
-			Resource: "images",
-		}, image.Name)
+		return nil, errors.NewAlreadyExists(api.Resource("images"), image.Name)
 	}
 
-	os.logger.Debug("(*FakeOpenShift).CreateImage: ", image.Name)
-	os.imagesStorage[image.Name] = image
+	fos.logger.Debug("(*FakeOpenShift).CreateImage: ", image.Name)
```

Comment:
Method logging is done within `fos.log`. This is logging of modification of `imagesStorage`/`imageStreamsStorage`. I changed a bit log message to make it clearer.